### PR TITLE
Add prefix-based SQS checks

### PR DESCRIPTION
**Pull request by reardencode**

### CODE REVIEW

File: bin/check-sqs-messages.rb [hunk 1]
```diff
@@ -96,14 +102,47 @@ def aws_config
   def run
     AWS.config aws_config
     sqs = AWS::SQS.new
-    messages = sqs.queues.named(config[:queue]).approximate_number_of_messages
+    if config[:prefix] == ''
+      if config[:queue] == ''
```
Comment (eheydrick) [inline bin/check-sqs-messages.rb]: could do `if config[:prefix] == '' && config[:queue] == ''` here

Comment (reardencode) [inline bin/check-sqs-messages.rb]: Sorta -- then I'd have to redo `if config[:prefix] == ""` again immediately after.

Comment (eheydrick) [inline bin/check-sqs-messages.rb]: ya, just saw that. nm

File: bin/check-sqs-messages.rb [hunk 2]
```diff
@@ -96,14 +102,47 @@ def aws_config
   def run
     AWS.config aws_config
     sqs = AWS::SQS.new
-    messages = sqs.queues.named(config[:queue]).approximate_number_of_messages
+    if config[:prefix] == ''
```
Comment (eheydrick) [inline bin/check-sqs-messages.rb]: add a blank line above this for readability.

File: bin/check-sqs-messages.rb [hunk 3]
```diff
@@ -56,7 +56,13 @@ class SQSMsgs < Sensu::Plugin::Check::CLI
          short: '-q SQS_QUEUE',
          long: '--queue SQS_QUEUE',
          description: 'The name of the SQS you want to check the number of messages for',
-         required: true
+         default: ''
+
+  option :prefix,
+         short: '-q PREFIX',
```
Comment (eheydrick) [inline bin/check-sqs-messages.rb]: need a unique option here. `-p` would be good.

File: bin/metrics-sqs.rb [hunk 4]
```diff
@@ -67,19 +73,38 @@ def aws_config
     }
   end
 
-  def run
-    scheme = if config[:scheme] == ''
-               "aws.sqs.queue.#{config[:queue].tr('-', '_')}.message_count"
```
Comment (eheydrick) [inline bin/metrics-sqs.rb]: need this .tr somewhere so the metric name stays the same.

Comment (reardencode) [inline bin/metrics-sqs.rb]: 497ce11

**Comment by eheydrick** [comment]
Thanks @reardencode!

**Pull request merged**